test/events/click.test.js

```diff
@@ -41,6 +41,45 @@ describe('click', function () {
 		destroy('config'); // Empty settings.
 	});
 
+	describe('when the old clicks queue exists', function() {
+		const clickEventStoredInQueue = {
+			"created_at": 1625589236422,
+			"item": {
+				"server": "https://spoor-api.ft.com/ingest",
+				"context": {
+					"product": "desktop",
+					"url": "https://www.example.com/",
+					"href": "https://www.example.com/",
+				},
+				"action": "click",
+				"category": "cta"
+			}
+		};
+		beforeEach(function() {
+			// Add the click event to the old 'clicks' queue which o-tracking v2 uses
+			new Queue('clicks').replace([clickEventStoredInQueue]);
+		});
+		beforeEach(function() {
+			// Remove the events from the old 'clicks' queue
+			new Queue('clicks').replace([]);
+		});
```

Comment:
I am a confused about why there are two `beforeEach` functions here and why the second doesn't 'overwrite' the output of the first?

Reply:
This was meant to be an afterEach, I'll update this now 👍